tests/clickhouse-test

```diff
@@ -1940,6 +1977,10 @@ class TestSuite:
                 )
 
             def filter_func(x: str) -> bool:
+                # Do not run this test in parallel batches,
+                # as it is designed to be executed in a single batch and may produce flaky results otherwise
+                if x == "00002_log_and_exception_messages_formatting.sql":
```

Comment:
Not sure how this works, but this test should be executed in all batches, at the end and serially. Now I don't see it anywhere (in the reports)

Reply:
This test is executed on single batch, so I decided to remove it from the CI multi-batch job. It still executes in the single-batch `Release` job.
The reason for this is that the test checks the threshold number of logs, and in batches, we execute far fewer tests, so fewer logs are generated. This causes the test to fail falsely due to the reduced log volume.

Reply:
This looks very icky in the code, maybe one more tag should be introduced to avoid this, like `no-batch` or similar

Reply:
@azat yes it's better, done